common-npm-packages/webdeployment-common/Tests/L0MSDeployUtility.ts

```diff
@@ -32,6 +32,18 @@ else {
     throw new Error('MSBUILD PACKAGE DEFAULT PARAMS FAILED');
 }
 
+var tokenAuthMSBuildPackageArgument: string = msdeployUtility.getMSDeployCmdArgs('package.zip', 'webapp_name', {
+    publishUrl: 'publishUrl', userName: 'user', userPWD: 'token'
+}, true, false, true, null, null, null, true, false, false, "Bearer");
```

Comment:
I can suggest using named arguments for readability, but it's not mandatory, so feel free to resolve this comment.

Reply:
How can we do that in TypeScript? I was under impression that there is not support for that.

Reply:
You're right